app/controllers/api/v1/deletions_controller.rb

```diff
@@ -46,4 +47,16 @@ def validate_gem_and_version
       end
     end
   end
+
+  def enqueue_web_hook_jobs
```

Comment:
Can we please move this and one used in Pusher to application controller? Call `enqueue_web_hook_jobs` after `gemcutter.process`.
https://github.com/rubygems/rubygems.org/blob/00468dc5fb20f6ae27c3d53528ed654b13cded51/app/models/pusher.rb#L126